pkg/vminfo/linux_syscalls.go

```diff
@@ -183,6 +184,13 @@ func linuxSyzKvmSetupCPUSupported(ctx *checkContext, call *prog.Syscall) string
 	return "unsupported arch"
 }
 
+func linuxSyzKvmVgicV3SetupSupported(ctx *checkContext, call *prog.Syscall) string {
+	if call.Name == "syz_kvm_vgic_v3_setup$arm64" && ctx.target.Arch == targets.ARM64 {
```

Comment:
VGIC is arm-specific, right?
https://docs.kernel.org/virt/kvm/devices/arm-vgic-v3.html

If so we could remove "$arm64" suffix and don't check the suffix here.

Reply:
Good idea.